### PR TITLE
Implements TEIIndexer.

### PR DESCRIPTION
**Pull request by cwulfman**
Advances #24

### CODE REVIEW

File: lib/tasks/cicognara.rake [hunk 1]
```diff
@@ -0,0 +1,12 @@
+require 'cicognara/tei_indexer'
+
+namespace :tei do
```
Comment (escowles) [inline lib/tasks/cicognara.rake]: Please rename the file to `tei.rake` to make it easier to figure out where the `tei:index` command is defined.

File: spec/cicognara/tei_indexer_spec.rb [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+      end
+
+      it 'dclib_s field to have 2 values when @corresp has 2 values' do
+        expect(subject.items[2].solr_doc[:dclib_s].length).to eq(2)
+      end
+    end
+  end
+
+  describe '#solr_docs' do
+    it 'has a length of 3' do
```
Comment (escowles) [inline spec/cicognara/tei_indexer_spec.rb]: These tests do a lot of type checking (which is fine), but I'd like to see some value tests too.  For example, you could get the first solr doc and check a couple of the values match what's in the fixture.

**Comment by escowles** [comment]
This looks like a good step towards getting the TEI indexed into Solr -- just a few comments I'd like addressed before this is ready to be merged!

**Comment by cwulfman** [comment]
@tampakis and I tried to figure out what happened with coveralls; will dig deeper tomorrow.

**Comment by escowles** [comment]
I'm not sure what's going on with Coveralls either.  It's not counting the User unit test (I even tried copying the tests to another file that was being counted).  So I wouldn't worry about it for now, as long as the functionality is working and there is decent test coverage.

**Comment by escowles** [comment]
👍

**Pull request merged**